### PR TITLE
[DE] Add `Gameplay/Play_time` and `Gameplay/Slider_break`

### PR DESCRIPTION
**Pull request by respektive**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Slider_break/de.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - combobreak
+  - combo-break
+  - SB
+---
+
+# Sliderbreak
+
+**Sliderbreak** (manchmal abgekürzt als *SB*) ist ein Begriff, der verwendet wird, um das Unterbrechen einer Combo durch einen teilweise vollendeten [Slider](/wiki/Hit_object/Slider) zu beschreiben. Sliderbreaks können auf drei verschiedene Arten auftreten: den Kopf eines Sliders zu verfehlen, während der Körper noch ganz oder teilweise fertiggestellt ist, einen Slider zu irgendeinem Zeitpunkt seiner Dauer loszulassen oder nicht alle Ticks des Sliders zu sammeln. Alle drei Arten werden zusammen als "Sliderbreak" bezeichnet und beenden die Combo des Spielers. Ein Sliderbreak während eines Durchgangs führt zu einem "100er" und unterbricht die Combo, zählt aber nicht als Miss.
+
+*Hinweis: Ein Durchgang, der keine Misses, aber einen oder mehrere Sliderbreaks enthält, führt zu einer S-Note (oder Silber S), vorausgesetzt, dass alle [Genauigkeit](/wiki/Gameplay/Accuracy)-Anforderungen erfüllt wurden, um eine S-Note zu erhalten.*
```
Comment (The-Last-Cookie) [inline wiki/Gameplay/Slider_break/de.md:17]:
```suggestion
*Hinweis: Ein Durchlauf, der keine Misses, aber einen oder mehrere Sliderbreaks enthält, führt zu einer S-Note (oder Silber S), vorausgesetzt, dass alle [Genauigkeits](/wiki/Gameplay/Accuracy)-Anforderungen erfüllt wurden, um eine S-Note zu erhalten.*
```

File: wiki/Gameplay/Slider_break/de.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - slider break
+  - combo break
+  - combobreak
+  - combo-break
+  - SB
+---
+
+# Sliderbreak
+
+**Sliderbreak** (manchmal abgekürzt als *SB*) ist ein Begriff, der verwendet wird, um das Unterbrechen einer Combo durch einen teilweise vollendeten [Slider](/wiki/Hit_object/Slider) zu beschreiben. Sliderbreaks können auf drei verschiedene Arten auftreten: den Kopf eines Sliders zu verfehlen, während der Körper noch ganz oder teilweise fertiggestellt ist, einen Slider zu irgendeinem Zeitpunkt seiner Dauer loszulassen oder nicht alle Ticks des Sliders zu sammeln. Alle drei Arten werden zusammen als "Sliderbreak" bezeichnet und beenden die Combo des Spielers. Ein Sliderbreak während eines Durchgangs führt zu einem "100er" und unterbricht die Combo, zählt aber nicht als Miss.
```
Comment (The-Last-Cookie) [inline wiki/Gameplay/Slider_break/de.md:15]:
```suggestion
**Sliderbreak** (manchmal abgekürzt als *SB*) ist ein Begriff, der verwendet wird, um das Unterbrechen einer Combo durch einen teilweise vollendeten [Slider](/wiki/Hit_object/Slider) zu beschreiben. Sliderbreaks können auf drei verschiedene Arten auftreten: den Kopf eines Sliders zu verfehlen, während der Körper noch ganz oder teilweise fertiggestellt ist, einen Slider zu irgendeinem Zeitpunkt seiner Dauer loszulassen oder nicht alle Ticks des Sliders zu sammeln. Alle drei Arten werden zusammen als "Sliderbreak" bezeichnet und beenden die Combo des Spielers. Ein Sliderbreak während eines Durchlaufs führt zu einem "100er" und unterbricht die Combo, zählt aber nicht als Miss.
```

Both is fine. `Durchlauf` sounds a little bit more common in this context in my opinion.

**Review by The-Last-Cookie — CHANGES_REQUESTED** [review]

File: wiki/Gameplay/Slider_break/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Gameplay/Slider_break/de.md:17]: `Genauigkeitsanforderungen` might also be possible.

Comment (respektive) [reply wiki/Gameplay/Slider_break/de.md:17]: i think this is good

**Review by TicClick — APPROVED** [review]

**Pull request merged**